libs/clj-icat-direct/src/clj_icat_direct/queries.clj

```diff
@@ -517,21 +517,19 @@
 
 (def queries
   {:count-all-items-under-folder
-   "WITH user_groups AS ( SELECT g.*
+   "WITH user_groups AS ( SELECT g.group_user_id
                             FROM r_user_main u
                             JOIN r_user_group g ON g.user_id = u.user_id
                            WHERE u.user_name = ?
                              AND u.zone_name = ? ),
 
-         parent      AS ( SELECT * from r_coll_main
+         parent      AS ( SELECT DISTINCT coll_id, coll_name from r_coll_main
```

Comment:
We should verify that. The same logic should have applied to the `SELECT count(DISTINCT c.coll_id)` in this query below, but when I tested it without the `DISTINCT` I got a higher count.